lib/features/drilldown/DrilldownOverlays.js

```diff
@@ -1,10 +1,13 @@
-import { domify } from 'min-dom';
+import { domify, classes } from 'min-dom';
 
 import { escapeHTML } from 'diagram-js/lib/util/EscapeUtil';
 import { getBusinessObject, is } from '../../util/ModelUtil';
 
 var ARROW_DOWN_SVG = '<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 16 16"><path fill-rule="evenodd" d="M4.81801948,3.50735931 L10.4996894,9.1896894 L10.5,4 L12,4 L12,12 L4,12 L4,10.5 L9.6896894,10.4996894 L3.75735931,4.56801948 C3.46446609,4.27512627 3.46446609,3.80025253 3.75735931,3.50735931 C4.05025253,3.21446609 4.52512627,3.21446609 4.81801948,3.50735931 Z"/></svg>';
 
+var HIDDEN_CLASS = 'djs-element-hidden',
+    OPEN_CLASS = 'bjs-breadcrumbs-open';
```

Comment:
We shall rename this to `shown`, to match our palette selectors, right? https://github.com/bpmn-io/diagram-js/pull/593

Reply:
We could also consider to rework the breadcrumbs to react directly to the `bjs-breadcrumbs-***` container class and remove the `HIDDEN_CLASS`.

Reply:
Right, I'll adjust this and the css selectors for the new diagram-js version :+1:

Reply:
Adjusted with d80d6492c113241f7b9c285fb40510de6653519c